sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/SmsClient.java

```diff
@@ -28,36 +25,35 @@
         this.smsAsyncClient = smsAsyncClient;
     }
 
-        /**
+
+    /**
      * Sends an SMS message from a phone number that belongs to the authenticated account.
      *
      * @param from Number that is sending the message.
-     * @param to The recipient's phone number. In this version, only one recipient in the list is supported.
+     * @param to The recipient's phone number.
      * @param message message to send to recipient.
-     * @param smsOptions set options on the SMS request, like enable delivery report, which sends a report
-     * for this message to the Azure Resource Event Grid.
-     * @param context the context of the request. Can also be null or Context.NONE.
      * @return response for a successful send Sms request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<SendSmsResponse> sendMessageWithResponse(PhoneNumberIdentifier from, 
-        List<PhoneNumberIdentifier> to, String message, 
-        SendSmsOptions smsOptions, Context context) {
-        return this.smsAsyncClient.sendMessageWithResponse(from, to, message, smsOptions, context).block();
+    public SmsSendResult send(String from, String to, String message) {
```

Comment:
All the Sync clients should have context passed in.

Reply:
In the arch Review a comment fron Srikanta was to add a Context only to the max overload of each method.